forage-android/src/main/java/com/joinforage/forage/android/LDManager.kt

```diff
@@ -85,4 +95,26 @@ internal object LDManager {
         internalLogger.i("[LaunchDarkly] Vault type set to $vaultType")
         return vaultType as VaultType
     }
+
+    internal fun getPollingIntervals(): IntArray {
```

Comment:
why do we return `IntArray` and then convert to long after getting the polling intervals?

Could we return a `LongArray`

Reply:
Yes, then we can get rid of the `toLong()` conversion when reading the value from the list